src/DacpacTool/PackageBuilder.cs

```diff
@@ -389,21 +389,21 @@ private List<int> ParseSuppressionList(string suppressionList)
             return result;
         }
 
-        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName, bool includeCompositeObjects)
+        public void GenerateCreateScript(BuildOptions options)
```

Comment:
I'd like to keep the seperation of concerns here and not pass in the `BuildOptions` directly, but keep the seperate parameters. I guess we could just replace the boolean with DacDeployOptions.

Reply:
Done. I agree it's better to separate the two concepts and apply the Hollywood principle :)